### ISSUE TITLE
Warn only once about bad edit note and break the mass click

### ISSUE THREAD

**Issue by jesus2099** [jesus2099/konami-command#476]
Not a new issue but it's true that it is done very crappy that it warns you for each step for each recording instead of only once and break the mass click (which is difficult to do, though).
https://community.metabrainz.org/t/merge-duplicate-recordings-between-two-editions-of-the-same-album-with-mb-mass-merge-recordings/203168/54?u=jesus2099

**Comment by psychoadept** [comment]
Just thought I'd mention this is still happening. Still obnoxious! Thanks.

**Comment by jesus2099** [comment]
Fixed by #712.
